rabbitmq/meta/collectd.yml

```diff
@@ -2,10 +2,11 @@
 local_plugin:
   rabbitmq_info:
     plugin: python
-    execution: local
     template: rabbitmq/files/collectd_rabbitmq_info.conf
-    username: {{ server.admin.name }}
-    password: {{ server.admin.password }}
+    host: {{ server.management.bind.address }}
+    port: {{ server.management.bind.port }}
+    username: guest
+    password: guest
```

Comment:
Right, I've been too hasty. But it seems that I can't authenticate with the admin credentials. I'll investigate this.